client/src/main/java/model/FleetMapper.java

```diff
@@ -4,19 +4,18 @@
 import java.util.List;
 
 /**
- * It maps the received Fleet to a fleet model.
+ * It maps the received FleetCreator to a fleet model.
  *
  * @author Patrycja Mikulska
  * @version 1.5
  */
 public class FleetMapper {
```

Comment:
Doesn't covered with unit test

Reply:
This is note my code, this will be coverd by Emilia in next pull request